war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -500,7 +500,11 @@ var tooltip;
 //========================================================
 // using tag names in CSS selector makes the processing faster
 function registerValidator(e) {
-    e.targetElement = findFollowingTR(e, "validation-error-area").firstChild.nextSibling;
+    var tr = findFollowingTR(e, "validation-error-area");
+    if (!tr || !tr.firstChild) {
```

Comment:
can we add a warning?
```suggestion
    if (!tr || !tr.firstChild) {
      console.warn("Couldn't register validator, start element was", e);
```

Reply:
Sure, I added a warning log.
The output is like this:
![warning](https://user-images.githubusercontent.com/12748956/89101771-1c88a700-d43e-11ea-8a91-764cb41af8b7.png)